pkg/function/restore_rds_snapshot.go

```diff
@@ -91,6 +91,7 @@ func (*restoreRDSSnapshotFunc) Arguments() []string {
 		RestoreRDSSnapshotUsername,
 		RestoreRDSSnapshotPassword,
 		RestoreRDSSnapshotNamespace,
+		RestoreRDSSnapshotSecGrpID,
```

Comment:
```suggestion
		RestoreRDSSnapshotSecGrpIDArg,
```
Should we add Arg suffix so there's no confusion about which consts are args?